System/Database/DatabaseInterface.py

```diff
@@ -58,19 +58,46 @@ def CheckIfUserExists(self, UserName:str): # Checks If A Given User Exists #
 
         # Execute Some MYSQL Cursor Code #
         # something here...
+        
+       
+        #Count of number of Users having the same UserName in the table
+        cnt= self.DatabaseCursor.execute("SELECT * from "+table+" where UserName='"+UserName+"'")
```

Comment:
If you use this it should just return a number:
"SELECT count(*) from..."

Reply:
Yes, the execute function returns the number of rows that are affected by the query.